core/io/file_access.h

```diff
@@ -247,8 +294,17 @@ class FileAccess : public RefCounted {
 	static bool get_read_only_attribute(const String &p_file);
 	static Error set_read_only_attribute(const String &p_file, bool p_ro);
 
-	static void set_backup_save(bool p_enable) { backup_save = p_enable; }
-	static bool is_backup_save_enabled() { return backup_save; }
+	static void set_backup_save(bool p_enable) {
+		WARN_DEPRECATED_MSG("Use set_default_integrity_level() instead.");
+		set_default_integrity_level(p_enable ? SAVE_INTEGRITY_SAVE_SWAP : SAVE_INTEGRITY_NONE);
+	}
```

Comment:
```suggestion
#ifndef DISABLE_DEPRECATED
	static void set_backup_save(bool p_enable) {
		WARN_DEPRECATED_MSG("Use set_default_integrity_level() instead.");
		set_default_integrity_level(p_enable ? SAVE_INTEGRITY_SAVE_SWAP : SAVE_INTEGRITY_NONE);
	}
#endif
```

Also, the same for the `OS::set_use_file_access_save_and_swap` in core bind.